src/pages/CropSelector/CropTable/CropTableCard.js

```diff
@@ -61,10 +51,7 @@ const CropTableCard = ({
         cropModifierAction(selectedCropsCopy, `${cropName} Removed`);
       }
     } else {
-      dispatch({
-        type: 'MY_CROP_LIST_LOCATION',
-        data: { from: 'selector' },
-      });
+      dispatchRedux(myCropListLocation({ from: 'explorer' }));
 
```

Comment:
I think this might cause problems since you change the 'from' property. 
This component is from the Listview of the Selector, now if you select any crops, it will show you the dialog warn you that you need to reset the list.